front/src/WebRtc/CoWebsiteManager.ts

```diff
@@ -91,6 +91,12 @@ class CoWebsiteManager {
             this.resizing = false;
         });
     }
+    
+    private getDevicePixelRatio(): number {
+        //on chrome, movementX and movementY return global screens coordinates while other browser return pixels
+        //so on chrome we need to adjust using 'devicePixelRatio'
+        return window.navigator.userAgent.includes('Chrome') ? window.devicePixelRatio : 1;
```

Comment:
I think Chrome / Chromium / Opera / Safari are using the same engine.
So we might fix the problem for Chrome but it might exist for other browsers.

You'd better test for Firefox instead.
```suggestion
        return !window.navigator.userAgent.includes('Firefox') ? window.devicePixelRatio : 1;
```

Reply:
According to this link: https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/screenX#browser_compatibility , only chrome and edge have the new behavior